cypress/integration/proposal-booking.spec.ts

```diff
@@ -0,0 +1,319 @@
+/// <reference types="Cypress" />
```

Comment:
I think you don't need this reference to types because you add "types": ["cypress"] to the tsconfig.json. Correct me if I am wrong.

Reply:
I am going to fix this on the user-office-frontend as well because I think it works perfectly to add types inside tsconfig.json file and remove them from each and every test file. You did a great job with this 💯

Reply:
Yes, indeed, it's no longer needed, nice catch